Source/RunActivity/Viewer3D/WebServices/Web/HUD/index.js

```diff
@@ -144,6 +150,7 @@ function ApiHeadUpDisplay() {
 							}
 							else if (ForceColSpanTo3.indexOf(obj.extraTable.values[next]) !== -1 ){
 								Str += "<td colspan='3' >" + obj.extraTable.values[next] + "</td>";
+								colspan2 = true; // data cells in these row span two columns
```

Comment:
For the rows that have per-axle values (in the Forces table).

Reply:
Agreed